doc/docs/master/how-tos/use-jupyterhub/index.md

```diff
@@ -0,0 +1,74 @@
+# Use JupyterHub with `python-pachyderm`
+
+This section describes how you can use the `python-pachyderm`
+client from within the JupyterHub UI.
+
+!!! note
+    You need to have Pachyderm and JupyterHub instlalled on the
+    same Kubernetes cluster as described in
+    [Deploy Pachyderm with JupyterHub](../../deploy-manage/deploy/deploy-pachyderm-jupyterhub.md).
+    Currently, an existing JupyterHub deployment cannot be
+    integrated with Pachyderm.
+
+## Overview
+
+JupyterHub is a popular data science platform that enables users
+to quickly spin out multiple single-tenant Jupyter Notebook server instances.
+Jupyter Notebook provides a Python interactive development environment (IDE)
+that is convenient for data science projects. Because of the built-in
+rich-text support, visualizations, the easy-to-use web interface, many
+enterprise users prefer Jupyter Notebooks to the classic Terminal prompt.
+JupyterHub brings all the benefits of Jupyter Notebooks without the need
+to install or configure anything on user machines except for a web browser.
+
+[python-pachyderm](https://github.com/pachyderm/python-pachyderm) is an
+official Python client for Pachyderm. For Python developers who prefer to
+communicate with Pachyderm directly through the API, rather than by using
+the `pachctl` tool, `python-pachyderm` is the right choice.
+The [API Documentation](https://pachyderm.github.io/python-pachyderm/python_pachyderm.m.html)
+describes various API operations that you can execute to interact with
+Pachyderm.
+
+When you deployed JupyterHub by using our deployment script, `python-pachyderm`
+was enabled in JupyterHub so that you can run API requests directly from
+your Jupyter Notebook.
+
+### Difference in Pipeline Creation Methods
+
+One of the most significant advantages of using `python-pachyderm`
+with JupyterHub is that you can avoid rebuilding your Docker images
+every time you update your code. Because JupyterHub is a Python environment,
+you can use the `create_python_pipeline` function to create a pipeline from
+within your JupyterHub Notebook and store your code locally rather than
+in a Docker container.
+
+When you use `pachctl` or the Pachyderm UI to create a pipeline, you execute
```

Comment:
This sentence is confusing. What about something like this?

```markdown
If you want to create a pipeline from a Docker image or non-Python code, you can use
[create_pipeline](https://pachyderm.github.io/python-pachyderm/python_pachyderm.m.html#python_pachyderm.Client.create_pipeline).
This function is akin to `pachctl create pipeline`.

Reply:
rephrased a bit.